pip install fails on macos

Trying to `pip install pyenet` fails on MacOS with
```
enet.c:785:10: fatal error: 'enet/types.h' file not found
```
I believe the issue is that the enet sources are not included in the tarball.

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -33,6 +33,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          submodules: true
 
       - name: Build sdist
         run: pipx run build --sdist

